### PR TITLE
WriteStats in prometheus handler

### PR DESCRIPTION
**Pull request by rohitggarg**
For those who are not using HTTP and still need to write stats through io.Writer

### CODE REVIEW

File: prometheus/handler.go [hunk 1]
```diff
@@ -135,9 +132,16 @@ func (h *Handler) ServeHTTP(res http.ResponseWriter, req *http.Request) {
 		w = zw
 	}
 
+	h.WriteStats(w)
+}
+
+func (h *Handler) WriteStats(w io.Writer) {
```
Comment (achille-roussel) [inline prometheus/handler.go]: Would you mind adding a bit of documentation to this method?

Comment (rohitggarg) [reply prometheus/handler.go]: Thanks!!
I will put some documentation tomorrow and do a bump

**Comment by rohitggarg** [comment]
Hi, I have added a basic doc. Is there like a guideline on how to contribute to documentation in this repository?

**Comment by achille-roussel** [comment]
We don't have guidelines, but your contribution looks great 👍 

Thanks for the PR, I'll go ahead and merge!

**Pull request merged**